### DISCUSSION TITLE
Numbers on X-axis instead of datetime

### DISCUSSION THREAD

**Discussion by IndiraPDG** [tradingview/lightweight-charts#1205] [Q&A] [answered]
Thank you for amazing library. Is there way I can format x-axis to have numbers instead of datetime as shown in the image?
![image](https://user-images.githubusercontent.com/70892939/203985512-1986b017-9133-407b-b752-ff8037c7a2fa.png)

Thanks!

**Comment by edew** [comment, marked as answer]
@IndiraPDG You could try something like this example, although the library is designed to have time values on the x axis:

https://codesandbox.io/s/number-time-axis-irscbt?file=/index.html

**Reply by IndiraPDG** [reply]
Thank you so much! It worked!
